routes/web.php

```diff
@@ -209,4 +210,16 @@
     Route::post('/community_service/reject/{community_service}', [CommunityServiceController::class, 'reject'])->name('community_service.reject');
     Route::post('/community_service/create', [CommunityServiceController::class, 'create'])->name('community_service.create');
     Route::get('/community_service/search', [CommunityServiceController::class, 'search'])->name('community_service.search');
+
+    Route::get('/voting', [VotingController::class, 'index'])->name('voting');
+    Route::get('/voting/new_sitting', [VotingController::class, 'newSitting'])->name('voting.new_sitting');
+    Route::post('/voting/add_sitting', [VotingController::class, 'addSitting'])->name('voting.add_sitting');
+    Route::get('/voting/{id}', [VotingController::class, 'viewSitting'])->name('voting.view_sitting');
+    Route::post('/voting/{id}/close', [VotingController::class, 'closeSitting'])->name('voting.close_sitting');
+    Route::get('/voting/{id}/new_question', [VotingController::class, 'newQuestion'])->name('voting.new_question');
+    Route::post('/voting/{id}/add_question', [VotingController::class, 'addQuestion'])->name('voting.add_question');
+    Route::post('/voting/questions/{id}/close', [VotingController::class, 'closeQuestion'])->name('voting.close_question');
+    Route::get('/voting/questions/{id}/vote', [VotingController::class, 'vote'])->name('voting.vote');
+    Route::post('/voting/questions/{id}/vote', [VotingController::class, 'saveVote'])->name('voting.vote.save');
+    Route::get('/voting/questions/{id}', [VotingController::class, 'viewQuestion'])->name('voting.view_question');
```

Comment:
```suggestion
    Route::get('/', [SittingController::class, 'index'])->name('sittings.index');
    Route::get('/sittings/create', [SittingController::class, 'createSitting'])->name('sittings.create');
    Route::post('/sittings', [SittingController::class, 'storeSitting'])->name('sittings.store');
    Route::get('/sittings/{sitting}', [SittingController::class, 'showSitting'])->name('sittings.show');
    Route::put('/sittings/{sitting}/close', [SittingController::class, 'closeSitting'])->name('sittings.close');
    Route::get('/voting/questions/create', [VotingController::class, 'createQuestion'])->name('voting.questions.create');
    Route::post('/voting/questions', [VotingController::class, 'storeQuestion'])->name('voting.questions.store');
    Route::get('/voting/questions/{questions}', [VotingController::class, 'showQuestion'])->name('voting.questions.show');
    Route::put('/voting/questions/{question}/close', [VotingController::class, 'closeQuestion'])->name('voting.questions.close');
    Route::get('/voting/questions/{questions}/votes/create', [VotingController::class, 'createVote'])->name('voting.questions.votes.create');
    Route::post('/voting/questions/{questions}/votes', [VotingController::class, 'storeVote'])->name('voting.questions.votes.store');

```

Reply:
Now here comes the fun part. Take a look at the CRUD (create-read-update-delete) concept here: https://laravel.com/docs/9.x/controllers#actions-handled-by-resource-controller
which gives us a nice structure for the whole resource handling. (We did not really use that in Mars yet)
If you want to follow this, you need to rename the links and controller functions (I suggest put it in a separate commit), but if you are in a hurry, don't worry, we will refactor later.

Reply:
Yes, I think I will take a look at it;)

Reply:
Done;) I removed the 'voting' prefixes from question-related routes; I think it is easier for now.